app.rb

```diff
@@ -27,18 +27,18 @@
   on post do
     on "create" do
       name = req.params["name"]
-      email = req.params["email"]
-      discord = req.params["discord"]
+      first-album-year = req.params["year"]
+      my-rank = req.params["rank"]
       db.execute(
-        "INSERT INTO students (name, email, discord) VALUES (?, ?, ?)",
-        name, email, discord
+        "INSERT INTO bands (name, firstalbumyear, myrank) VALUES (?, ?, ?)",
+        name, firstalbumyear, myrank
```

Comment:
This would work if your variables above were named `firstalbumyear` and `myrank`. 

If you didn't run into any bugs when running your code, it may have been because you didn't try creating a new band!